src/core/Akka.Remote/RemoteSystemDaemon.cs

```diff
@@ -124,7 +125,53 @@ protected override void TellInternal(object message, IActorRef sender)
                 Log.Debug("Received command [{0}] to RemoteSystemDaemon on [{1}]", message, Path.Address);
                 if (message is DaemonMsgCreate) HandleDaemonMsgCreate((DaemonMsgCreate)message);
             }
+            else if (message is ActorSelectionMessage sel)
```

Comment:
Most of the heavy lifting for #1544 occurs here inside the `RemoteDaemon` - we never programmed it to handle `ActorSelectionMessage` instances before. 

Opted not to use `ValueTuple` here since we're adding that dependency in 1.4. Didn't want to introduce it now for such a small change.